src/i18n/locales/de-DE.json

```
@@ -0,0 +1,773 @@
+    "Wavelength": "Wellenlänge",
+    "Woodland": "Wald"
+  },
+  "calling_card_categories": {
+    "campaign": "Kampagne",
+  "multiplayer": "Mehrspieler",
+  "dmz": "DMZ",
+  "dmz_activate_uav": "UAVs aktivieren",
+  "dmz_agent_kills": "Agentenabschüsse",
+  "dmz_armored_agent_kills": "Gepanzerte Agentenabschüsse",
+  "dmz_clear_strongholds": "Festungen säubern",
+  "dmz_complete_contracts": "Verträge abschließen",
+  "dmz_defeat_bosses": "Bosse besiegen",
+  "dmz_exfil": "Exfiltration",
+  "dmz_exfil_cash": "Exfiltrationsgeld",
+  "dmz_exfil_final_helicopter": "Endhubschrauber für die Exfiltration",
+  "dmz_extract_valuables": "Wertgegenstände extrahieren",
+  "dmz_loot_caches": "Beutekisten plündern",
+  "dmz_operator_kills": "Operatorabschüsse",
+  "dmz_unlock_spaces": "Räume freischalten",
+  "multiplayer_assault_kills": "Angriffsabschüsse",
+  "multiplayer_defender_kills": "Verteidigungsabschüsse",
+  "multiplayer_misc": "Sonstiges",
+  "multiplayer_occupying_vehicles": "Besetzte Fahrzeuge",
+  "multiplayer_wins": "Gewinne",
+  "operator": "Operator",
+  "operator_assists": "Assists",
+  "operator_double_kills": "Doppelte Abschüsse",
+  "operator_finishing_moves": "Abschlussbewegungen",
+  "operator_headshots": "Kopfschüsse",
+  "operator_hipfire_kills": "Hüftfeuerabschüsse",
+  "operator_kills": "Abschüsse",
+  "operator_kills_lethals": "Abschüsse mit tödlichen Waffen",
+  "operator_longshots": "Fernschüsse",
+  "operator_melee": "Nahkampfabschüsse",
+  "operator_without_dying": "Abschüsse ohne zu sterben",
+  "prestige 1": "Prestige 1",
+  "prestige 2": "Prestige 2",
+  "prestige 3": "Prestige 3",
+  "prestige 4": "Prestige 4",
+  "prestige 5": "Prestige 5",
+  "prestige 6": "Prestige 6",
+  "prestige 7": "Prestige 7",
+  "prestige 8": "Prestige 8",
+  "prestige 9": "Prestige 9",
+  "prestige 10": "Prestige 10",
+  "prestige 11": "Prestige 11",
+  "prestige 12": "Prestige 12",
+  "prestige 13": "Prestige 13",
+  "prestige 14": "Prestige 14",
+  "prestige 15": "Prestige 15",
+  "prestige 16": "Prestige 16",
+  "prestige 17": "Prestige 17",
+  "prestige 18": "Prestige 18",
+  "prestige 19": "Prestige 19",
+  "prestige 20": "Prestige 20",
+  "prestige 21": "Prestige 21",
+  "prestige 22": "Prestige 22",
+  "prestige 23": "Prestige 23",
+  "prestige 24": "Prestige 24",
+  "prestige 25": "Prestige 25",
+  "season": "Saison",
+  "twitch": "Twitch",
+  "warzone": "Warzone",
+  "warzone_containers": "Container",
+  "warzone_contracts": "Verträge",
+  "warzone_kills": "Abschüsse",
+  "warzone_wins": "Warzone Siege"
+  },
+  "weapon_categories": {
+    "Assault Rifles": "Sturmgewehre",
+    "Battle Rifles": "Kampfgewehre",
+    "Handguns": "Pistolen",
+    "Launchers": "Werfer",
+    "Light Machine Guns": "Leichte Maschinengewehre",
+    "Marksman Rifles": "DMR",
+    "Melee": "Nahkampf",
+    "Shotguns": "Schrotflinten",
+    "Sniper Rifles": "Scharfschützengewehre",
+    "Sub Machine Guns": "Maschinenpistolen"
+  }
+}
```

There seems to be a missing end `}` somewhere in the structure, but I can't find it. Just add one to the end and it should hopefully solve itself.
```suggestion
}
}
```